conans/client/migrations.py

```diff
@@ -63,7 +63,7 @@ def _make_migrations(self, old_version):
 # Only for building cross compilation tools, 'os_target/arch_target' is the system for
 # which the tools generate code
 os_target: [Windows, Linux, Macos, Android, iOS, watchOS, tvOS, FreeBSD, SunOS, Arduino]
-arch_target: [x86, x86_64, ppc32, ppc64le, ppc64, armv6, armv7, armv7hf, armv7s, armv7k, armv8, armv8_32, armv8.3, sparc, sparcv9, mips, mips64, avr]
```

Comment:
Remove all this, the migrations file only have to contain the previous setting.yml, otherwise the migration will fail.

Reply:
removed